experiments/ClimaEarth/leaderboard/leaderboard.jl

```diff
@@ -0,0 +1,168 @@
+import ClimaAnalysis
+import ClimaUtilities.ClimaArtifacts: @clima_artifact
+import GeoMakie
+import CairoMakie
+import Dates
+
+include("data_sources.jl")
+
+# leaderboard_base_path is the path to save the leaderboards
+# diagnostics_folder_path is the path to simulation data
+"""
+    compute_leaderboard(leaderboard_base_path, diagnostics_folder_path)
+
+Plot the biases and a leaderboard of various variables.
+
+The paramter `leaderboard_base_path` is the path to save the leaderboards and bias plots and
+`diagnostics_folder_path` is the path to the simulation data.
+"""
+function compute_leaderboard(leaderboard_base_path, diagnostics_folder_path)
+    @info "Error against observations"
+
+    # Set up dict for storing simulation and observational data after processing
+    sim_obs_comparsion_dict = Dict()
+    seasons = ["ANN", "MAM", "JJA", "SON", "DJF"]
+
+    # Print dates for debugging
+    pr_var = sim_var_dict["pr"]() # it shouldn't matter what short name we use
```

Comment:
Instead of assuming that `pr` is available, you can pick the first entry in the dict